documentation/examples/15-composition/05-modal/Modal.svelte

```diff
@@ -6,13 +6,13 @@
 	$: if (dialog && showModal) dialog.showModal();
 </script>
 
-<!-- svelte-ignore a11y-click-events-have-key-events -->
+<!-- svelte-ignore a11y-click-events-have-key-events a11y-no-noninteractive-element-interactions -->
 <dialog
 	bind:this={dialog}
 	on:close={() => (showModal = false)}
 	on:click|self={() => dialog.close()}
 >
-	<div on:click|stopPropagation>
+	<div on:click|stopPropagation role='button' tabindex='0'>
```

Comment:
This shouldn't be made a button - the click handler is just listening to all child events. In this case it's okay to ignore the warning since screen reader users have other ways to close a native dialog and don't need to use this click handler.